README.md

```diff
@@ -1,2 +1,91 @@
 # brainmatch
-Event project requirement-contributor request matching toolkit
+Event project requirement-contributor request matching toolkit.
+
+## Introduction
+
+The purpose of these scripts is to find matches between projects submitted to
+a Brainhack Global event and the participants in that event. The match works
+by scoring the matches between a project's requirements (e.g. topic, required
+skills or tools) and the participant's skills or background and interests.
+
+Once the result is generated, the Brainhack Global event organizers will be
+able to send participants an email with the project information that
+potentially fit them better within the context of the event.
+
+## Context
+
+The tools assume that the projects submitted as issues to the [https://github.com/brainhackorg/global2020](https://github.com/brainhackorg/global2020)
+repository have been labelled correctly, meaning that they have been assigned
+to a given Brainhack Global event, that they have been already published to the
+website, and they contain the appropriate labels concerning their topic and
+required skills and tools.
+
+Similarly, it assumed that your Brainhack Global event participant data is
+available as a `CSV` file, and that for each participant it contains the data
+that will allow matching them to a project. That is, it will contain
+information about the participants' skills or background and interests.
+
+Note that in order for the scripts to work, the relevant participant data with
+respect to the projects needs to match exactly with the available labels in the
+project data in the [https://github.com/brainhackorg/global2020](https://github.com/brainhackorg/global2020) repository.
+
+The scoring method does not currently take into account the required level of
+expertise for a project, nor is the participant's desired project type taken
+into account.
+
+## Instructions
+
+In order to obtain the project-participant matches, you will need to:
+
+1. Pull the project data locally, by calling the `tools/pull_issues.sh` script.
+The script will output a `TSV` file containing all relevant data from all
+existing projects in the [https://github.com/brainhackorg/global2020](https://github.com/brainhackorg/global2020)
+issues.
+
+1. Your registration form is likely to use some custom text to gather the
+required participant information. These data are expected to be readable as
+separate data pieces whose headings or titles can be matched to some standard
+fields used in the scripts so that the appropriate data can be retrieved for
+finding the matches.
```

Comment:
This sentence sounds a bit complicated. Maybe splitting it into two sentences improves readability.

Reply:
Done in 3038215.